conan/tools/microsoft/visual.py

```diff
@@ -56,7 +57,20 @@ def msvc_version_to_toolset_version(version):
                 '191': 'v141',
                 '192': 'v142',
                 "193": 'v143'}
-    return toolsets[str(version)]
+    return toolsets.get(str(version))
+
+
+def visual_version_to_toolset_version(version):
```

Comment:
Leave this an internal implementation detail of ``msvs_toolset``, not even as a separate function.

Reply:
Done.